source/OptimaJet.DWKit.StarterApplication/Models/IStatusUpdate.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace OptimaJet.DWKit.StarterApplication.Models
+{
+    public interface IStatusUpdate
```

Comment:
Originally, `IStatusUpdate` just had `UpdateGroups` to know which groups should be notified.  Then I had to add StringId since some Model objects might inherit from either `Identifiable` or `Identifiable<Guid>` and `StringId` was common to both interfaces.  So I added it to `IStatusUpdate`.  At another time, it had `EntryEntity` so that we could get the `TableName` when the implementation was in `AppDbContext`.  Since we moved it to the Repository implementation, we hav to lookup the TableName from the contained `DbContext`.

So IStatusUpdate is not a create name.  We need some way to mark a model object so that we know that we want to send out notifications (because it isn't free).

Reply:
I found I could use `IIdentifiable` so I don't need `IStatusUpdate` or `IEntityHook` (or any additional interface to mark which to check).  Now everything is done through DI.